api/src/main/resources/workbench-api.yaml

```diff
@@ -350,6 +350,33 @@ paths:
           description: Bad request
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/link-ras-account":
+    post:
+      tags:
+      - profile
+      consumes:
+      - application/json
+      description: Link user with a RAS account
+      operationId: linkRasAccount
+      parameters:
+      - in: body
```

Comment:
use request body instead of body because I remembered some swagger codegen client(jersey2?) does not allow empty request body? 
And that follows other requests.

Reply:
Is this relevant?  It seems that we need to use a body to contain the Era Commons name regardless

Reply:
I was trying to explain why not put authcode in body, not query like Bond does:
https://github.com/DataBiosphere/bond/blob/develop/swagger/api-docs.yaml#L158

Reply:
body looks fine to me